packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3482,7 +3489,37 @@ export default class Meeting extends StatelessWebexPlugin {
       // for multistream WCME automatically un-publishes the old track when we publish a new one
       await this.unpublishTrack(oldTrack);
     }
-    await this.publishTrack(this.mediaProperties.shareTrack);
+    await this.publishTrack(this.mediaProperties.shareVideoTrack);
+  }
+
+  /**
+   * Stores the reference to a new screen share audio track, sets up the required event listeners
+   * on it, cleans up previous track, etc.
+   * It also sends the floor grant/release request.
+   *
+   * @param {LocalSystemAudioTrack | undefined} localSystemAudioTrack local system audio track
+   * @returns {Promise<void>}
+   */
+  private async setLocalShareAudioTrack(localSystemAudioTrack?: LocalSystemAudioTrack) {
+    const oldTrack = this.mediaProperties.shareAudioTrack;
+
+    oldTrack?.off(LocalTrackEvents.Ended, this.handleShareAudioTrackEnded);
+    oldTrack?.off(LocalTrackEvents.UnderlyingTrackChange, this.underlyingLocalTrackChangeHandler);
+
+    this.mediaProperties.setLocalShareAudioTrack(localSystemAudioTrack);
+
+    localSystemAudioTrack?.on(LocalTrackEvents.Ended, this.handleShareAudioTrackEnded);
+    localSystemAudioTrack?.on(
+      LocalTrackEvents.UnderlyingTrackChange,
+      this.underlyingLocalTrackChangeHandler
+    );
+
+    this.mediaProperties.mediaDirection.sendShare = !!localSystemAudioTrack;
```

Comment:
this should be `!!this.mediaProperties.shareVideoTrack || !!this.mediaProperties.shareAudioTrack` and same in setLocalShareVideoTrack()

Reply:
fixed this